### PR TITLE
{bio}[foss/2023a] phyluce v1.7.3

### PR DESCRIPTION
**Pull request by pavelToman**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ generoso

**Comment by pavelToman** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/p/phyluce/phyluce-1.7.3_removal-Bio-Alphabet.patch [hunk 1]
```diff
@@ -0,0 +1,22 @@
+removal of the Bio.Alphabet - it is not part of Biopython anymore (since v1.78)
+see also https://biopython.org/wiki/Alphabet
+author: Pavel Tomanek (INUITS)
```
Comment (boegel) [inline easybuild/easyconfigs/p/phyluce/phyluce-1.7.3_removal-Bio-Alphabet.patch:3]: @pavelToman I made a small change here, by adding you as author of this patch (and then changing the SHA256 checksum for this patch accordingly in the easyconfig)

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/t/trimAl/trimAl-1.4.1-GCCcore-12.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+builddependencies = [
+    ('binutils', '2.40'),
+
+]
+start_dir = 'source'
+
+files_to_copy = [(['trimal', 'readal', 'statal'], 'bin')]
+
+sanity_check_paths = {
+    'files': ['bin/trimal', 'bin/readal', 'bin/statal'],
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/t/trimAl/trimAl-1.4.1-GCCcore-12.3.0.eb:33]: fixed in [c43ff54](https://github.com/easybuilders/easybuild-easyconfigs/pull/19779/commits/c43ff54c33d75d50967054da9c438293f105312d)

File: easybuild/easyconfigs/p/Pilon/Pilon-1.23-Java-11-wrapped.eb [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    'files': ['pilon-%(version)s.jar'],
+    'dirs': [],
+}
+
+postinstallcmds = [
+    "cd %(installdir)s && ln -s %(namelower)s-%(version)s.jar pilon.jar",
+    "mkdir -p %(installdir)s/bin",
+    "echo '#!/bin/sh' > %(installdir)s/bin/%(namelower)s",
+    "echo 'java -jar -Xmx8G %(installdir)s/%(namelower)s-%(version)s.jar $@' >> %(installdir)s/bin/%(namelower)s",
+    "chmod a+rx %(installdir)s/bin/%(namelower)s"
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/p/Pilon/Pilon-1.23-Java-11-wrapped.eb]: fixed in [f6e5f3f](https://github.com/easybuilders/easybuild-easyconfigs/pull/19779/commits/f6e5f3f2724377900796b1626575d3299ce641d7)

File: easybuild/easyconfigs/p/Pilon/Pilon-1.23-Java-11.eb [hunk 4]
```diff
@@ -2,29 +2,37 @@ easyblock = 'JAR'
 
 name = 'Pilon'
 version = '1.23'
-versionsuffix = '-Java-%(javaver)s'
+versionsuffix = '-Java-%(javaver)s-wrapped'
```
Comment (boegel) [inline easybuild/easyconfigs/p/Pilon/Pilon-1.23-Java-11.eb:5]:
```suggestion
versionsuffix = '-Java-%(javaver)s'
```

**Review by boegel — CHANGES_REQUESTED** [review]
fix versionsuffix for Pilon

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 9 out of 9 (9 easyconfigs in total)
node3120.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/eed8d5f60988771f57452ba85b63a34c for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Going in, thanks @pavelToman!

**Pull request merged**